dipper/sources/AnimalQTLdb.py

```diff
@@ -337,8 +337,8 @@ def _process_QTLs_genetic_location(
                     reference = Reference(g, pub_id)
                 elif pubmed_id != '':
                     pub_id = 'PMID:'+pubmed_id.strip()
-                    reference = Reference(g,
-                        pub_id, Reference.ref_types['journal_article'])
+                    reference = Reference(
+                        g, pub_id, Reference.ref_types['journal_article'])
```

Comment:
In the spirit of the o->common_name refactor above, change g to graph.

Reply:
I can't even guess what 'k' is ... maybe 'kind'?